app/models/journal.rb

```diff
@@ -1,2 +1,4 @@
 class Journal < ApplicationRecord
+  belongs_to :user
+  validates_presence_of(:title, :date, :content, :goals_tomorrow, :goals_today, message: ->(_, _) { I18n.t('validations.presence') })
```

Comment:
Let's remove the i18n and re-write the validations separately so that we can handle the individual validations later on when required.
```
validates :title, presence: true
validates :date, presence: true
```
and so on...

Regarding the i18n, rails will throw the default translation for this validation

Reply:
Let's also add db level validations after confirming the model level validations